pkg/operations/kubernetesupgrade/upgradecluster_test.go

```diff
@@ -197,29 +195,6 @@ var _ = Describe("Upgrade Kubernetes cluster tests", func() {
 		Expect(err.Error()).To(Equal("DeleteNetworkInterface failed"))
 	})
 
```

Comment:
This was testing for aks-engine downgrade validation, which is now handled at the cmd layer.